tests/integration/stepfunctions/conftest.py

```diff
@@ -0,0 +1,213 @@
+import logging
+from typing import Final
+
+import pytest
+
+from localstack.aws.api.stepfunctions import ExecutionStatus
+from localstack.utils.strings import short_uid
+from localstack.utils.sync import poll_condition
+
+LOG = logging.getLogger(__name__)
+
+# For EXPRESS state machines, the deletion will happen eventually (usually less than a minute).
+# Running executions may emit logs after DeleteStateMachine API is called.
+_DELETION_TIMEOUT_SECS: Final[int] = 120
+
+
+@pytest.fixture
+def snf_snapshot(snapshot):
+    return snapshot
+
+
+@pytest.fixture
+def create_iam_role_for_snf(create_iam_role_with_policy):
```

Comment:
```suggestion
def create_iam_role_for_sfn(create_iam_role_with_policy):
```
 Also some `anr` => `arn`.
There are a few small typos like this through the PR. :